### ISSUE TITLE
CKV2_AWS_65 false positive on aws_s3_bucket_ownership_controls

### ISSUE THREAD

**Issue by wahlfeld** [bridgecrewio/checkov#5623]
**Describe the issue**
CKV2_AWS_65 was introduced [last week](https://github.com/bridgecrewio/checkov/pull/5599) and correctly detected an ACL in my Terraform code, however, it's also failing against a "aws_s3_bucket_ownership_controls" resource. 

The "aws_s3_bucket_ownership_controls" resource is not the same as ACLs. Ownership controls are a separate feature in S3 that allow you to manage ownership of new objects. They don't provide the same kind of access control that ACLs or IAM policies do.

**Examples**
```
Check: CKV2_AWS_65: "Ensure access control lists for S3 buckets are disabled"
        FAILED for resource: module.main.aws_s3_bucket_ownership_controls.bla
        File: /module/s3.tf:19-25

                19 | resource "aws_s3_bucket_ownership_controls" "bla" {
                20 |   bucket = aws_s3_bucket.bla.id
                21 | 
                22 |   rule {
                23 |     object_ownership = "BucketOwnerPreferred"
                24 |   }
                25 | }
```

**Version (please complete the following information):**
v.2.5.6

**Comment by gruebel** [comment]
hey @wahlfeld thanks for reaching out.

I also thought so, but the docs tell a different story.

https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html
> Bucket owner enforced (default) – ACLs are disabled, and the bucket owner automatically owns and has full control over every object in the bucket. ACLs no longer affect permissions to data in the S3 bucket. The bucket uses policies to define access control.

So, I interpret it as disabling ACLs completely and shift access controls to policy level.

**Comment by wahlfeld** [comment]
Thanks for getting back to me. I interpret the docs differently. To me it reads as:

_Bucket owner enforced (default) - When this setting is **on** then ACLs **are/become** disabled._

It goes on to say _"ACLs **no longer** affect permissions to data in the S3 bucket"_, or in other words, when bucket owner enforced is enabled it _means_ ACLs "no longer" have no effect on the permissions of the bucket. This is different to what the Checkov rule is saying, which is that ownership controls are similar to ACLs in that AWS recommends disabling both in favour of bucket policies. 

Hope that makes sense! Not sure if you have TAM or contact in the S3 team you can confirm this with?

**Comment by gruebel** [comment]
Found some other docs, which make it quite clear

https://docs.aws.amazon.com/AmazonS3/latest/userguide/ensure-object-ownership.html
> We recommend that you disable ACLs on your Amazon S3 buckets. You can do this by applying the Bucket owner enforced setting for S3 Object Ownership. When you apply this setting, ACLs are disabled and you automatically own and have full control over all objects in your bucket.

I'm fine with adjusting the check name, but your suggestion sounds somehow misleading, because it doesn't disable object ownership. How about `Ensure ACLs for S3 buckets are disabled and enforce object ownership`?
